src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -93,21 +56,30 @@ public EntityAIStructureBuilder(@NotNull final JobBuilder job)
     {
         super(job);
         super.registerTargets(
+          /**
+           * If IDLE - switch to start working.
+           */
+          new AITarget(IDLE, START_WORKING),
           new AITarget(this::checkIfCanceled, IDLE),
           new AITarget(this::checkIfExecute, this::getState),
-          new AITarget(IDLE, START_WORKING),
-          new AITarget(START_WORKING, this::startWorkingAtOwnBuilding),
-          new AITarget(BUILDER_CLEAR_STEP, this::clearStep),
-          new AITarget(BUILDER_REQUEST_MATERIALS, this::requestMaterials),
-          new AITarget(BUILDER_STRUCTURE_STEP, this::structureStep),
-          new AITarget(BUILDER_DECORATION_STEP, this::decorationStep),
-          new AITarget(BUILDER_COMPLETE_BUILD, this::completeBuild)
+          new AITarget(START_WORKING, this::startWorkingAtOwnBuilding)
         );
         worker.setSkillModifier(INTELLIGENCE_MULTIPLIER * worker.getCitizenData().getIntelligence()
                                   + STRENGTH_MULTIPLIER * worker.getCitizenData().getStrength());
         worker.setCanPickUpLoot(true);
     }
 
+    @Override
+    public Block getSolidSubstitution(BlockPos location)
+    {
+        IBlockState filler = world.getBiome(location).fillerBlock;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'filler' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)